src/utils/imageCompress.ts

```diff
@@ -0,0 +1,8 @@
+export enum AvailableForCompressExtention {
+    PNG = 'png',
+    JPG = 'jpg',
+    JPEG = 'jpeg',
+}
+
+export const canBeCompressed = (fileName: string) =>
+    Object.keys(AvailableForCompressExtention).find((ext) => fileName.endsWith(`.${ext}`));
```

Comment:
`.some` work similarly like `.find`, but always return `boolean`. If we need boolean in  [disableWebp](https://github.com/gravity-ui/page-constructor/pull/215/files#diff-da73b37ebbe2383290a1d6caf9037c4598e9d500d0632596c8dc877a191ad10fR40) variable always, we need to use `.some`

Reply:
ok